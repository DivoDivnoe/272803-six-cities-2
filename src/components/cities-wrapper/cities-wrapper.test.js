import React from 'react';
import renderer from 'react-test-renderer';
import CitiesWrapper from './cities-wrapper.jsx';

describe(`MainPage component`, () => {
  it(`is rendered correctly`, () => {
    const offers = 1;

    const tree = renderer
      .create(<CitiesWrapper offers={offers} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
