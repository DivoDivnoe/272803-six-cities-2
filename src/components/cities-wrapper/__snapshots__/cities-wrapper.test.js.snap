// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage component is rendered correctly 1`] = `
<div
  className="cities"
>
  <div
    className="cities__places-container container "
  />
</div>
`;
